src/trees/MWTree.cpp

```diff
@@ -133,8 +133,16 @@ template <int D> void MWTree<D>::mwTransformDown(bool overwrite) {
             int n_nodes = nodeTable[n].size();
 #pragma omp for schedule(guided)
             for (int i = 0; i < n_nodes; i++) {
+                int reverse = 7;
```

Comment:
This looks specific for 3D?

Reply:
think i fixed it, I moved it into the negative scale test, then replaced it with `int reverse = n_nodes - 1`